testsuite/features/secondary/proxy_as_pod_basic_tests.feature

```diff
@@ -0,0 +1,298 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# The scenarios in this feature are skipped if:
+# * there is no proxy ($proxy is nil)
+# * there is no salt minion ($sle_minion is nil)
+# * there is no scope @scope_containerized_proxy
+
+@scope_containerized_proxy
+@proxy
+@sle_minion
+Feature: Register and test a Containerized Proxy
+  In order to test Containerized Proxy
+  As the system administrator
+  I want to register the proxy to the server
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Pre-requisite: Unregister Salt minion in the traditional proxy
+    Given I am on the Systems overview page of this "sle_minion"
+    When I stop salt-minion on "sle_minion"
+    And I follow "Delete System"
+    Then I should see a "Confirm System Profile Deletion" text
+    When I click on "Delete Profile"
+    Then I wait until I see "Cleanup timed out. Please check if the machine is reachable." text
+    When I click on "Delete Profile Without Cleanup" in "An error occurred during cleanup" modal
+    And I wait until I see "has been deleted" text
+    Then "sle_minion" should not be registered
+
+  Scenario: Pre-requisite: Stop traditional proxy service
+    When I stop salt-minion on "proxy"
+    And I run "spacewalk-proxy stop" on "proxy"
+    And I wait until "squid" service is inactive on "proxy"
+    And I wait until "apache2" service is inactive on "proxy"
+    And I wait until "jabberd" service is inactive on "proxy"
+
+  Scenario: Generate Containerized Proxy configuration
+    When I generate the configuration "/tmp/proxy_container_config.zip" of Containerized Proxy on the server
+    And I copy "/tmp/proxy_container_config.zip" file from "server" to "proxy"
+    And I run "unzip -qq -o /tmp/proxy_container_config.zip -d /etc/uyuni/proxy/" on "proxy"
+
+  Scenario: Set the Containerized Proxy to use specific container images
+    When I set a new value in a configuration file
```

Comment:
I am no big fan of this table syntax. I don't find it very legible. But that's probably a matter of tastes.

Reply:
Yup, it's just that the line becomes hyper-long otherwise.
The good part of this, is that for RC1, we might use already the default values of that configuration file.
In that case, I will remove this step. And then, we might think if we want to keep this step even if not used, or as you promote apply YAGNI, and remove unused code.

But for now.. commiting.